### PR TITLE
refactor: Clean after code review

### PR DESCRIPTION
**Pull request by serereg**
Clean after code review.

Relates to https://github.com/epam/badgerdoc/issues/66
Relates to https://github.com/epam/badgerdoc/issues/170

### CODE REVIEW

File: convert/tests/test_labelstudio/test_import_text.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    def mock_download_text(*args, **kwargs):
+        return "1"
+
+    def mock_upload_text(*args, **kwargs):
+        pass
+
+    monkeypatch.setattr(
+        TextToBadgerdocConverter, "download", mock_download_text
+    )
+    monkeypatch.setattr(TextToBadgerdocConverter, "upload", mock_upload_text)
```
Comment (cakeinsauce) [inline convert/tests/test_labelstudio/test_import_text.py:25]: Up to you

```suggestion
    monkeypatch.setattr(
        TextToBadgerdocConverter, "download", lambda *args, **kw: "1"
    )
    monkeypatch.setattr(TextToBadgerdocConverter, "upload",  lambda *args, **kw: pass)
```

File: convert/tests/test_labelstudio/test_import_labelstudio.py [hunk 2]
```diff
@@ -2,41 +2,90 @@
...
+        mock_download_labelstudio,
+    )
+    monkeypatch.setattr(
+        LabelstudioToBadgerdocConverter,
+        "upload",
+        mock_upload_text,
+    )
+    monkeypatch.setattr(
+        LabelstudioToBadgerdocConverter, "execute", mock_execute
+    )
```
Comment (cakeinsauce) [inline convert/tests/test_labelstudio/test_import_labelstudio.py:69]: Up to you
```suggestion
    monkeypatch.setattr(
        LabelstudioToBadgerdocConverter,
        "download",
        lambda *args, **kw: LabelStudioModel(),
    )
    monkeypatch.setattr(
        LabelstudioToBadgerdocConverter,
        "upload",
        lambda *args, **kw: pass,
    )
    monkeypatch.setattr(
        LabelstudioToBadgerdocConverter, "execute", lambda *args, **kw: pass
    )
```

File: convert/tests/test_labelstudio/test_export_labelstudio.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+            "path": "test_converter/out.json",
+        },
+    }
+
+    def mock_execute(*args, **kwargs):
+        pass
+
+    monkeypatch.setattr(
+        BadgerdocToLabelstudioConverter, "execute", mock_execute
+    )
```
Comment (cakeinsauce) [inline convert/tests/test_labelstudio/test_export_labelstudio.py:45]: Up to you
```suggestion
    monkeypatch.setattr(
        BadgerdocToLabelstudioConverter, "execute", lambda *args, **kw: pass
    )
```

File: convert/tests/test_labelstudio/conftest.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        tenants=["tenant", "test"],
+    )
+    return mock_tenant_data
+
+
+@pytest.fixture
+def test_app(setup_tenant):
+    main.app.dependency_overrides[labelstudio.tenant] = lambda: setup_tenant
+    client = TestClient(main.app)
+    yield client
```
Comment (cakeinsauce) [inline convert/tests/test_labelstudio/conftest.py:33]:
```suggestion
    yield TestClient(main.app)
```

File: convert/tests/test_labelstudio/conftest.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+@pytest.fixture()
+def setup_tenant():
+    mock_tenant_data = TenantData(
+        token="token",
+        user_id="owner1",
+        roles=["admin"],
+        tenants=["tenant", "test"],
+    )
+    return mock_tenant_data
```
Comment (cakeinsauce) [inline convert/tests/test_labelstudio/conftest.py:26]:
```suggestion
    return TenantData(
        token="token",
        user_id="owner1",
        roles=["admin"],
        tenants=["tenant", "test"],
    )
```

File: convert/tests/test_labelstudio/test_export_labelstudio.py [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+from src.labelstudio_format import annotation_converter_practic
+from src.labelstudio_format.ls_format import LabelStudioFormat
+from src.labelstudio_format.ls_models import LabelStudioModel
+
+TEST_FILES_DIR = Path(__file__).parent / "test_data"
+
+INPUT_LABELSTUDIO_FILE = TEST_FILES_DIR / "labelstudio_format.json"
+
+
+def test_correctness_of_export_text_schema(test_app, monkeypatch):
```
Comment (cakeinsauce) [inline convert/tests/test_labelstudio/test_export_labelstudio.py:23]: Add typings, please

File: convert/tests/test_labelstudio/test_export_labelstudio.py [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+    response = test_app.post(
+        "/labelstudio/export",
+        json=test_request_payload,
+    )
+
+    assert response.status_code == 201
+
+
+@responses.activate
+def test_annotation_converter_case_without_taxonomies_and_document_labels():
```
Comment (cakeinsauce) [inline convert/tests/test_labelstudio/test_export_labelstudio.py:56]: Add typings, please

File: convert/tests/test_labelstudio/test_import_labelstudio.py [hunk 8]
```diff
@@ -2,41 +2,90 @@
...
 
-INPUT_LABEL_STUDIO_FILE = TEST_FILES_DIR / "input_text_field.json"
-INPUT_LABELSTUDIO_FILE = TEST_FILES_DIR / "label_studio_format.json"
+INPUT_labelstudio_FILE = TEST_FILES_DIR / "input_text_field.json"
+INPUT_LABELSTUDIO_FILE = TEST_FILES_DIR / "labelstudio_format.json"
 BADGERDOC_TOKENS_FILE = TEST_FILES_DIR / "badgerdoc_tokens.json"
 TEST_PDF = TEST_FILES_DIR / "test.pdf"
 
 
+def test_correctness_of_import_text_schema(test_app, monkeypatch):
```
Comment (cakeinsauce) [inline convert/tests/test_labelstudio/test_import_labelstudio.py:32]: Add typings, please

File: convert/tests/test_labelstudio/test_import_labelstudio.py [hunk 9]
```diff
@@ -2,41 +2,90 @@
...
+
+    response = test_app.post(
+        "/labelstudio/import",
+        json=test_request_payload,
+    )
+
+    assert response.status_code == 201
+
+
 def test_plain_text_converter():
```
Comment (cakeinsauce) [inline convert/tests/test_labelstudio/test_import_labelstudio.py:79]: Add typings, please

File: convert/tests/test_labelstudio/test_import_labelstudio.py [hunk 10]
```diff
@@ -86,17 +134,16 @@ def test_annotation_converter():
 
 
 def test_import_document_links():
```
Comment (cakeinsauce) [inline convert/tests/test_labelstudio/test_import_labelstudio.py:136]: Add typings, please

File: convert/tests/test_labelstudio/test_import_text.py [hunk 11]
```diff
@@ -0,0 +1,36 @@
+import pytest
+
+from src.plain_text_format.plain_text_converter import (
+    TextToBadgerdocTokensConverter,
+)
+from src.text_to_badgerdoc_converter import TextToBadgerdocConverter
+
+
+def test_correctness_of_import_text_schema(test_app, monkeypatch):
```
Comment (cakeinsauce) [inline convert/tests/test_labelstudio/test_import_text.py:9]: Add typings, please

File: convert/src/plain_text_format/plain_text_converter.py [hunk 12]
```diff
@@ -118,24 +125,35 @@ def line_tokens_amount(self) -> int:
         token_size = self.font_width
         return (self.page_width - self.page_border_offset * 2) // token_size
 
-    def convert_line(self, line, tokens, y, line_offset: int):
-        x = self.page_border_offset
+    def convert_line(self, line, tokens, y_axis, line_offset: int):
```
Comment (cakeinsauce) [inline convert/src/plain_text_format/plain_text_converter.py:128]: Add typings, please

File: convert/src/plain_text_format/plain_text_converter.py [hunk 13]
```diff
@@ -118,24 +125,35 @@ def line_tokens_amount(self) -> int:
...
                 offset=offset,
             )
             tokens.append(token)
-            x += self.font_width
+            x_axis += self.font_width
 
+    @staticmethod
     def calculate_page_size(
-        self, tokens: List[BadgerdocToken], page_border_offset
+        tokens: List[BadgerdocToken], page_border_offset
```
Comment (cakeinsauce) [inline convert/src/plain_text_format/plain_text_converter.py:149]: Add typings, please

**Review by cakeinsauce — CHANGES_REQUESTED** [review]
Few more things:
- You don't need to add empty `__init__.py` in every folder. Python has namespace packages
- Let's make a package called `formats` which contains subpackages with different convert models. And remove format word from anywhere here, please. E.g. `formats` [package]  -> `badgerdoc` [package] -> `class Badgerdoc`

File: convert/src/text_to_badgerdoc_converter.py [hunk 14]
```diff
@@ -52,10 +54,10 @@ def download_text_from_s3(
             )
             return input_file.read_text()
 
-    def upload_badgerdoc_to_s3(
+    def upload(
         self,
-        s3_output_tokens,
-        s3_output_pdf,
+            s3_output_tokens: S3Path,
```
Comment (cakeinsauce) [inline convert/src/text_to_badgerdoc_converter.py:59]: Remove redundant tab please

File: convert/src/badgerdoc_to_labelstudio_converter.py [hunk 15]
```diff
@@ -103,17 +104,17 @@ def download_file_from_s3(self, s3_path: S3Path, tmp_dir: Path) -> Path:
         )
         return local_file_path
 
-    def upload_labelstudio_to_s3(
+    def upload(
         self,
         s3_output_annotation: S3Path,
     ):
```
Comment (cakeinsauce) [inline convert/src/badgerdoc_to_labelstudio_converter.py:110]: add typing, please

File: convert/tests/test_labelstudio/test_import_text.py (hunk 11 above)
Comment (serereg) [reply convert/tests/test_labelstudio/test_import_text.py]: Done

File: convert/tests/test_labelstudio/conftest.py [hunk 16]
```diff
@@ -0,0 +1,33 @@
...
+
+
+@pytest.fixture
+def test_app(setup_tenant):
```
[the remark this answers is not in the document]
Comment (serereg) [reply convert/tests/test_labelstudio/conftest.py]: Done

File: convert/tests/test_labelstudio/conftest.py [hunk 17]
```diff
@@ -0,0 +1,33 @@
...
+
+class TenantData(BaseModel):
+    token: str
+    user_id: str
+    roles: List[str]
+    tenants: List[str]
+
+
+@pytest.fixture()
+def setup_tenant():
```
[the remark this answers is not in the document]
Comment (serereg) [reply convert/tests/test_labelstudio/conftest.py]: Done

File: convert/src/text_to_badgerdoc_converter.py [hunk 18]
```diff
@@ -1,16 +1,16 @@
...
     TextToBadgerdocTokensConverter,
 )
-from .models.common import S3Path
 
 
-class TextToBDConvertUseCase:
+class TextToBadgerdocConverter:
     def __init__(
         self,
         s3_client,
```
[the remark this answers is not in the document]
Comment (serereg) [reply convert/src/text_to_badgerdoc_converter.py]: Done

File: convert/src/plain_text_format/plain_text_converter.py (hunk 12 above)
Comment (serereg) [reply convert/src/plain_text_format/plain_text_converter.py]: Done

File: convert/src/text_to_badgerdoc_converter.py (hunk 14 above)
Comment (serereg) [reply convert/src/text_to_badgerdoc_converter.py]: Done

File: convert/src/badgerdoc_to_labelstudio_converter.py (hunk 15 above)
Comment (serereg) [reply convert/src/badgerdoc_to_labelstudio_converter.py]: Done

File: convert/tests/test_labelstudio/test_import_labelstudio.py (hunk 8 above)
Comment (serereg) [reply convert/tests/test_labelstudio/test_import_labelstudio.py]: Done

File: convert/tests/test_labelstudio/test_export_labelstudio.py (hunk 6 above)
Comment (serereg) [reply convert/tests/test_labelstudio/test_export_labelstudio.py]: Done

File: convert/tests/test_labelstudio/test_import_labelstudio.py (hunk 8 above)
Comment (cakeinsauce) [reply convert/tests/test_labelstudio/test_import_labelstudio.py]: Return typing as well, please

Comment (serereg) [reply convert/tests/test_labelstudio/test_import_labelstudio.py]: Done

**Review by cakeinsauce — APPROVED** [review]
LGTM!
Squash commits before merging, please
And don't forget about repos' commits convention

**Pull request merged**